web-api/src/net.rs

```diff
@@ -93,9 +95,16 @@ pub(crate) fn start_actix_server(
                     .route(web::get().to(HttpResponse::Ok))
                     .wrap(Cors::default()),
             )
+            .service(
+                web::scope("/_ops")
+                    .configure(&attach_state)
+                    .configure(&attach_ops)
+                    .wrap(Cors::default()),
```

Comment:
This changes `/_silo_management` => `/_ops/silo_management`.

The problem is that a wild card scope `scope("")` is too greedy and will prevent routing the normal app.
Similar only ingestion has `_silo_management` but this is code used by both. Lastly we might have other
ad-hoc operations/management endpoints in the future and would have the same problem. So by having
a `_ops` scope we don't have the issue. I already informed ops.